sklearn/tests/test_multiclass.py

```diff
@@ -470,16 +482,28 @@ def test_ovo_partial_fit_predict():
     assert_almost_equal(pred1, pred2)
 
     # Test when mini-batches don't have all target classes
-    ovo1 = OneVsOneClassifier(MultinomialNB())
-    ovo1.partial_fit(iris.data[:60], iris.target[:60], np.unique(iris.target))
-    ovo1.partial_fit(iris.data[60:], iris.target[60:])
-    pred1 = ovo1.predict(iris.data)
-    ovo2 = OneVsOneClassifier(MultinomialNB())
-    pred2 = ovo2.fit(iris.data, iris.target).predict(iris.data)
+    p = digits.target.argsort()
+    X, y = digits.data[p], digits.target[p]
+    n_classes_digits = len(np.unique(y))
 
-    assert_almost_equal(pred1, pred2)
-    assert_equal(len(ovo1.estimators_), len(np.unique(iris.target)))
-    assert_greater(np.mean(iris.target == pred1), 0.65)
+    ovo = OneVsOneClassifier(MultinomialNB())
+    ovo.partial_fit(X[:500], y[:500], np.unique(y))
+    ovo.partial_fit(X[500:], y[500:])
+    pred = ovo.predict(X)
+    ovr2 = OneVsOneClassifier(MultinomialNB())
+    pred2 = ovr2.fit(X, y).predict(X)
+
+    assert_almost_equal(pred, pred2)
+    assert_equal(len(ovo.estimators_), n_classes_digits * (n_classes_digits - 1) / 2)
+    assert_greater(np.mean(y == pred), 0.65)
+
+    # This test fails for OvR, issue #6202 needs to be solved
+    ovo = OneVsOneClassifier(SGDClassifier())
+    X = np.random.randn(10, 2)
+    ovo.partial_fit(X[:5], [1, 1, 2, 1, 1], [0, 1, 2])
+    ovo.partial_fit(X[5:], [0, 0, 0, 0, 0])
```

Comment:
hmm. In order to avoid being dependent on the other issue, I would suggest to increase the number of classes in the first pass to 4 and in the second to 2.